### DISCUSSION TITLE
I have 3 4 Repos  that have more than 16 star but then also i dint got the star badge ?

### DISCUSSION THREAD

**Discussion by awesome-yasin** [community/community#36251] [Other Feature Feedback, Questions, & Ideas]
I have 3 4 Repos  that have more than 16 star but then also i dint got the star badge. one repos has 25 another has 23 and one has 75 stars but i got star only on one repo. why ???

**Comment by HalemoGPA** [comment]
### **But you already got it!** 
![image](https://user-images.githubusercontent.com/73307941/195995455-1adc1019-b975-4ed3-8a25-77cd4ae9dd6c.png)

**Reply by BaoTrung37** [reply]
Your rep needs 128 stars to unlock next level.

![image](https://user-images.githubusercontent.com/61321804/195995549-a03c6682-d2ba-49d6-8593-a947db95d482.png)

**Comment by JestherCostinar** [comment]
You need at least 128 stars to a single repository.

reference: https://github.com/sagelga/github-achievement-cheatsheet

**Reply by ghulamrasool154** [reply]
ok thanks
